src/kbemu.c

```diff
@@ -37,26 +39,13 @@ KeySym kbemu_keysym(const char *str)
 
 void send_kbevent(KeySym key, int press)
 {
-	XEvent xevent;
-	Window win;
-	int rev_state;
-
 	if(!dpy) return;
 
+	Window win;
+	int rev_state;
```

Comment:
Similarly since this is a C89 codebase, variable declarations need to be at the top of the enclosing scope.